.github/actions/README.md

```
@@ -13,6 +13,19 @@ executable bits or `./script.sh` invocation.
 
 ### Core CI/CD Actions
 
+#### `go-test/`
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)